### PR TITLE
MGMT-13704: Adapt signatures to get host logs both with new and old paths

### PR DESCRIPTION
**Pull request by nmagnezi**
While we transition to the new logs path structure (see [MGMT-13454](https://issues.redhat.com/browse/MGMT-13454)), signatures
should search for logs both in the old and the new location (in case the
former failed).

Once [MGMT-13454](https://issues.redhat.com/browse/MGMT-13704) lands in production and is confirmed to be working as
expected, we will remove the old path from the signatures code base.

### CODE REVIEW

**Comment by omertuc** [comment]
ctrl+f `logs_host_` I think some other signatures need fixing

**Comment by nmagnezi** [comment]
> ctrl+f `logs_host_` I think some other signatures need fixing

Done

**Review by eranco74 — COMMENTED** [review]
Overall looks good, just a nit

**Comment by nmagnezi** [comment]
> Overall looks good, just a nit

@eranco74, can't see the comment. Is it pushed?

**Comment by osherdp** [comment]
Can you add more description in the PR? I think it's missing some context

**Comment by nmagnezi** [comment]
> Can you add more description in the PR (& commit message)? I think it's missing some context

@osherdp Done.

File: tools/add_triage_signature.py [hunk 1]
```diff
@@ -201,13 +201,29 @@ def get_host_log_file(triage_logs_tar, host_id, filename):
...
+
+    return agent_logs
 
 
 def get_journal(triage_logs_tar, host_ip, journal_file):
-    return triage_logs_tar.get(
-        f"*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/control-plane/{host_ip}/journals/{journal_file}"
-    )
+    new_logs_path = f"*_bootstrap_*.tar/*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/control-plane/{host_ip}/journals/{journal_file}"
+    old_logs_path = f"*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/control-plane/{host_ip}/journals/{journal_file}"
```
Comment (eranco74) [inline tools/add_triage_signature.py:218]: Consider having a const for the new and old logs_path prefix
e.g.
```
new_logs_path_prefix = f"*_bootstrap_*.tar/*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/"
old_logs_path_prefix = f"*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/"
 ```
 and use that everywhere,

Comment (nmagnezi) [reply tools/add_triage_signature.py]: Done

**Review by filanov — APPROVED** [review]
Could use join for the paths, but it's a minor comment.

**Pull request merged**